deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -92,6 +92,12 @@ spec:
             mountPath: /available-configs
           - name: config
             mountPath: /config
+        {{- with .Values.resources }}
```

Comment:
A general question: Does it make sense to apply the same resources to each of the containers?